example/index.js

```diff
@@ -1,19 +1,28 @@
+import Web3 from "web3"
+import web3Modal from './providers.js'
+
+let web3
```

Comment:
Why do you import web3 here? We only need the provider right?

Reply:
I just tested with removing web3 and it works just fine.

Reply:
wasnt sure if all the providers alone had the same interface for getting accounts, but yeah may confusing anyone reading 👍  although example not most friendly code example already